synapse/models/material.py

```diff
@@ -31,37 +31,95 @@ class MatModule(s_module.CoreModule):
 
     def getModelDefs(self):
         modl = {
+
+            'interfaces': (
+
+                ('phys:object', {
+                    'doc': 'Properties common to all physical objects.',
+                    'template': {'phys:object': 'object'},
+                    'props': (
+
+                        ('phys:mass', ('mass', {}), {
+                            'doc': 'The mass of the {phys:object}.'}),
+
+                        ('phys:volume', ('geo:dist', {}), {
+                            'doc': 'The cubed volume of the {phys:object}.'}),
+
+                        ('phys:length', ('geo:dist', {}), {
+                            'doc': 'The length of the {phys:object}.'}),
+
+                        ('phys:width', ('geo:dist', {}), {
+                            'doc': 'The width of the {phys:object}.'}),
+
+                        ('phys:height', ('geo:dist', {}), {
+                            'doc': 'The height of the {phys:object}.'}),
+                    ),
+                }),
+            ),
+
             'types': (
-                ('mat:item', ('guid', {}), {'doc': 'A GUID assigned to a material object.'}),
+
+                ('phys:object', ('ndef', {'interface': 'phys:object'}), {
+                    'doc': 'A node which represents a physical object.'}),
+
+                ('phys:contained:type:taxonomy', ('taxonomy', {}), {
+                    'interfaces': ('meta:taxonomy',),
+                    'doc': 'A taxonomy for types of contained relationships.'}),
```

Comment:
An example (`'ex': '...'`} might be nice here. Based on `phys:contained` description, it seems like this should be something like `liquid` or `grains.large` or something like that?